dlc-manager/src/manager.rs

```diff
@@ -37,16 +37,54 @@ use std::collections::HashMap;
 use std::ops::Deref;
 use std::string::ToString;
 
-/// The number of confirmations required before moving the the confirmed state.
+/// The number of btc confirmations required before moving the DLC to the confirmed state.
 pub const NB_CONFIRMATIONS: u32 = 6;
-/// The delay to set the refund value to.
-pub const REFUND_DELAY: u32 = 86400 * 7;
+/// The minimum time to trigger the refund.
+pub const REFUND_DELAY_MIN: u32 = 86400 * 7;
+/// The maximum time to trigger the refund.
+pub const REFUND_DELAY_MAX: u32 = 86400 * 7 * 2;
 /// The nSequence value used for CETs in DLC channels
 pub const CET_NSEQUENCE: u32 = 288;
 /// Timeout in seconds when waiting for a peer's reply, after which a DLC channel
 /// is forced closed.
 pub const PEER_TIMEOUT: u64 = 3600;
 
+/// The options used to configure the DLC manager.
+pub struct ManagerOptions {
+    /// The number of btc confirmations required before moving the DLC to the confirmed state.
+    pub nb_confirmations: u32,
+    /// The refund delay window to trigger the refund.
+    pub refund_delay: RefundDelayWindow,
+    /// The nSequence value used for CETs in DLC channels
+    pub cet_nsequence: u32,
+    /// Timeout in seconds when waiting for a peer's reply, after which a DLC channel
+    /// is forced closed.
+    pub peer_timeout: u64,
+}
+
+/// The min and max value in seconds until a DLC will be refunded.
+#[derive(Clone, Copy)]
+pub struct RefundDelayWindow {
```

Comment:
Looking through the code I'm not sure this is really useful, in some places you end up passing that when only one of the value is required.